build-llvm.py

```diff
@@ -104,6 +104,14 @@ def parse_parameters(root_folder):
                         By default, the script always updates the LLVM repo before building. This prevents
                         that, which can be helpful during something like bisecting.
 
+                        """),
+                        action="store_true")
+    parser.add_argument("-o",
```

Comment:
bikeshed on naming; `-o` is usually "what to call the output," so using it to enable an option violates the principle of least surprise.  Please just call the option `--march-native` or something of the like.

Reply:
Sure

Reply:
Done.